src/components/header/header.jsx

```diff
@@ -4,10 +4,13 @@ import { useTheme } from '../../context/theme/theme-context';
 import { Link, NavLink } from 'react-router-dom';
 import '../css/header.css';
 import { useWishlist } from '../../context/wishlist/wishlist-context';
+import { useMyCart } from '../../context/mycart/mycart-context';
```

Comment:
suggestion: you can use an index file in context and export these files from there so that it is cleaner at import side

Reply:
Yes, will be doing that. Thanks